### DISCUSSION TITLE
Guidance on Cloning Vidstack Examples from GitHub

### DISCUSSION THREAD

**Discussion by byte-lingual** [vidstack/player#977] [General]
Can someone direct me on how to clone and load the vidstack examples from https://github.com/vidstack/examples? I tried connecting to it from my VScode terminal and it said I didn't have permission?... I don't have much experience with VScode connecting to github and even more much experience with how github works and it's terminology. Are there any good books one can recommend as well that teaches the workflow/ culture of github? Thanks

**Comment by mihar-22** [comment]
You can simply write Git tutorial in Google or YouTube and start learning. The simpler option for you is to open the example you want in StackBlitz. Go to the example directory you want and click the blue button that says "Open in StackBlitz".
